init_pose:
  position: [-0.954769432545, 1.48137366772, 0.0]
  orientation: [0.0, 0.0, -0.694655700189, 0.719342378979]
